Cargo.toml

```diff
@@ -31,10 +31,12 @@ edition = "2021"
 
 [workspace.dependencies]
 anyhow = "1.0.98"
+borsh = "0.10.4"
```

Comment:
The version of borsh synced to the version used in `mpl-token-metadata` due to compatibility reasons